tests/untldoc_test.py

```diff
@@ -712,3 +712,34 @@ def test_find_untl_errors_fix_errors_no_errors():
                                         'date': [{'content': '1944',
                                                   'qualifier': 'creation'}]},
                           'error_dict': {}}
+
+
+def test_untl_to_hash_dict():
+    title = us.Title(qualifier='serialtitle', content='The Bronco')
+    meta_modifier = us.Meta(qualifier='metadataModifier', content='Daniel')
+    meta_modification = us.Meta(qualifier='metadataModificationDate',
+                                content='2007-09-20, 13:46:15')
+    meta_object = us.Meta(qualifier='objectType', content='simple')
+    elements = us.Metadata()
+    elements.add_child(title)
+    elements.add_child(meta_modification)
+    elements.add_child(meta_modifier)
+    elements.add_child(meta_object)
+    hash_dict = untldoc.untl_to_hash_dict(elements)
+    assert hash_dict == {'title': '9eff715f7ee7da9d5c2efdf075d07225',
+                         'meta': 'f66a40a765dbfa230bd1b250a465ff6d'}
+
+
+def test_untl_dict_to_tuple():
+    untl_dict = {'title': [{'qualifier': 'serialtitle',
```

Comment:
I suggest also changing to `untl_dict = {'creator': [{'qualifier': 'edt',`, so that the element content makes sense in the context of untl elements.